lib/src/arithmetic/values/floating_point_values/floating_point_value_populator.dart

```diff
@@ -206,6 +245,11 @@ class FloatingPointValuePopulator<FpvType extends FloatingPointValue> {
   FpvType ofDouble(double inDouble,
       {FloatingPointRoundingMode roundingMode =
           FloatingPointRoundingMode.roundNearestEven}) {
+    if (explicitJBit) {
+      return toExplicitJBitFloatingPointValue(FloatingPointValue.populator(
```

Comment:
Shouldn't this be using `_unpopulated.clonePopulator()` instead of the base class one in case some type has its own populator?